Assets/Editor/ModelInspector.cs

```diff
@@ -20,26 +20,32 @@ internal static class Styles
             public static GUIStyle preSliderThumb = "preSliderThumb";
         }
 
-        private PreviewRenderUtility m_PreviewUtility;
+        internal struct PreviewSettings
+        {
+            public DisplayMode displayMode;
+            public int activeUVChannel;
+            public bool drawWire;
+
+            public Vector3 orthoPosition;
+            public Vector2 previewDir;
+            public float zoomFactor;
+            public int checkerTextureMultiplier;
+
+            public Material shadedPreviewMaterial;
+            public Material activeMaterial;
+            public Material meshMultiPreviewMaterial;
+            public Material wireMaterial;
+            public Material lineMaterial;
+            
+            public bool[] availableDisplayModes;
+            public bool[] availableUVChannels;
+        }
         
-        private Material m_ShadedPreviewMaterial;
-        private static Material m_MeshMultiPreviewMaterial;
-        private Material m_WireMaterial;
-        private static Material m_LineMaterial;
-
-        private Material m_activeMaterial;
+        private PreviewRenderUtility m_PreviewUtility;
+        private PreviewSettings m_ModelInspectorSettings;
```

Comment:
Minor, for shorter names: `m_ModelInspectorSettings` could be changed to `m_Settings`, and `previewSettings` could be changed to `settings` everywhere in this PR, without loss of clarity I think.